tools/js2c.py

```diff
@@ -293,16 +293,18 @@ def JS2C(source, target):
     lines = ExpandMacros(lines, macros)
     lines = CompressScript(lines, do_jsmin)
     data = ToCArray(s, lines)
-    id = os.path.basename(str(s)).split('.')[0]
+    id = '/'.join((str(s).split(os.sep)[1:])).split('.')[0]
```

Comment:
I'm not 100% sure if you should be using `os.sep` here... I guess it depends on whether gyp converts the slashes to backslashes on Windows.